definitions/variable/power_plant.yaml

```
@@ -0,0 +1,10 @@
+# List of variables related to power plant attributes
+- Inflows|Electricity|Hydro:
+    description: (energy-equivalent) inflows of water into hydro storages
+    unit: GWh
+- Inflows|Electricity|Hydro|Pumped:
+    description: (energy-equivalent) inflows of water into reservoirs of pumped hydro
+    unit: GWh
+- Inflows|Electricity|Hydro|Hydro:
```

It's not clear to me what you mean by "Hydro|Hydro" - I guess it's reservoirs that do not have a pump storage unit?
```suggestion
- Inflows|Electricity|Hydro|Reservoir:
```

---

Yes, it's supposed to be "Reservoir". Thanks for the review!